source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
+                     </specList></p>
+                     <p>The <gi scheme="MEI">dimensions</gi> contain information about the physical size of a source. Usually the dimensions are represented by numerical data. The elements <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, <gi scheme="MEI">depth</gi>, and <gi scheme="MEI">dim</gi> are available for circumstances that require the capture of individual dimensions. To indicate the quantity of the described materials, please refer to the extent element.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample086.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="dim"/>
+                     </specList></p>
+                     <p>Another way of encoding is to use the element <gi scheme="MEI">dim</gi>, which can be used to represent any dimensional specification in a generic way. The attribute <att>form</att> is required. With regard to structured metadata, the use of the <gi scheme="MEI">height</gi>, <gi scheme="MEI">width</gi>, and <gi scheme="MEI">depth</gi> elements as the content of <gi scheme="MEI">dim</gi> is advisable.</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample087.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p><specList>
+                        <specDesc key="watermark"/>
+                     </specList></p>
+                     <p> Before starting to describe and encode watermarks, it is highly recommended to read up on filigranology separately. It is important to distinguish whether the watermark to be described consists of only one sign or is composed of main and countermarks. The completeness of the watermark, its positioning on the manuscript page, and the time period of the paper's production and use also play a role.</p>
+                     <p>It is recommended to use at least the elements <gi scheme="MEI">title</gi>, <gi scheme="MEI">date</gi>, <gi scheme="MEI">annot</gi> and <gi scheme="MEI">locus</gi> in the description:</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample080.txt" parse="text"/></egXML>
```

Comment:
The new sample080.txt is overwriting the existing sample for the scoreFormat that is used later on (line 1528).

Reply:
This has to be carefully addressed when resolving conflicts and merging.

Reply:
@riedde I checked with your ig-metadata branch. There the `scoreFormat` sample file has the number 94.